src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -1134,6 +1134,7 @@ public void testContainerWithHostConfig() throws Exception {
         .publishAllPorts(publishAllPorts)
         .dns(dns)
         .cpuShares((long) 4096)
+        .cpuQuota((long) 50000)
```

Comment:
For this test to be meaningful, we should inspect the container and see this value right? I don't know why we didn't do it for cpuShares above.